### DISCUSSION TITLE
Excluding specific Home Assistant entries from Google Home

### DISCUSSION THREAD

**Discussion by Zombie-aaargh** [Luligu/matterbridge#117] [General]
Is there a way to exclude specific entities - As Home assistant calls them?

I have a Synology NAS that has it's own internal temperature readings, and when I use Matterbridge these are picked up by Google Home and included alongside my regular Temperature and humidity sensors.

For example, I have these three lines i my available devices section of the web UI

Plugin name 	Device type 	Endpoint 	Name 	Serial number 	Unique ID 	Cluster 

matterbridge-home-assistant	MA-tempsensor (0x0302)	189	Average disk temp	(number)	(number) Temperature: 0°C

matterbridge-home-assistant	MA-tempsensor (0x0302)	190	Temperature	(number) (number) Temperature: 0°C

matterbridge-home-assistant	MA-tempsensor (0x0302)	191	Temperature	(number) (number) Temperature: 0°C

How would I go about excluding these so that Google doesn't pick them up, I'd like to keep the overall class of entity, and to include some of the other entities so I don't wan to exclude the entire device?

**Comment by Luligu** [comment]
Hi,
ask there matterbridge-home-assistant.
My best
